addons/website/static/src/snippets/s_image_gallery/001.scss

```diff
@@ -141,24 +141,34 @@
         }
         &.s_image_gallery_indicators_arrows_boxed, &.s_image_gallery_indicators_arrows_rounded {
             .carousel {
-                .carousel-control-prev .fa,
-                .carousel-control-next .fa {
+                .carousel-control-prev .fa, .oi,
+                .carousel-control-next .fa, .oi {
```

Comment:
Hi @divy-odoo have you seen that you put the same selector twice in several places?

Reply:
oh, my bad! 
@bvr-odoo it's fixed now.